### DISCUSSION TITLE
How set batch size in track.py?

### DISCUSSION THREAD

**Discussion by kodinkod** [mikel-brostrom/yolo_tracking#930] [Q&A] [answered]
how i may process video faster

**Comment by mikel-brostrom** [comment, marked as answer]
I guess this could be done. But the bottleneck is the reid model, not yolo. I will try to explain why

1. Yolo input: (bs, h,w,3)
2. Yolo output after NMS: [[N, 6], [M, 6], ..., [X, 6]]
Each of these output are used to crop each of the original input images. All the crops are then stacked together for each image and fed to the ReID model
4. ReID input : (N, h, w, 3), (M, h, w, 3), ...
The dimensions of a conv network are maximum 4. So you would have to process the images sequentially anyways for the ReID model.  Not so simple as just increasing the batch size for yolo.
